docs/DesignDocs/SGX_QuoteEx_Integration.md

```diff
@@ -197,16 +206,28 @@ It first calls into the quote-ex library to check if the dependent background
 service is available.
 If so, the quote-ex library is used, otherwise the DCAP library is used.
 
+#### Proposal: Built-time Link with the quote-ex Library
+
+The proposal is to implement option 3, to support build-time link with only
```

Comment:
Per the [SIG-Attestation meeting](https://hackmd.io/Xj6GpDSKSwuz5cZgQ0yg1A#13-May-2020), we shouldn't break the existing in-proc loading behavior of the existing `oe_get_report()` function until we deprecate that in favor of the extensible `oe_get_evidence()` APIs. This would be pending Intel's release of documentation for dealing with the `sgx_prv` security group with the upstream kernel driver as well.

In the meantime, we should support both and verify that:
- We can link both DCAP and quote-ex libraries to by default in the OE SDK cmake export at the same time.
- Callers of `oe_get_report()` continue to be able to load the QE in-proc, while the callers of `oe_get_evidence()` get the new behavior through loading quote-ex and using the out-of-proc AESM hosted QE when both libraries are linked. 

As a follow-up during implementation, can you also work with SIG-testing to ensure that we have the new dependencies available in the testing pipeline?

Reply:
Agreed that we should keep the existing in-proc behavior. Will update the proposal section. Will work with SIG-testing when implementation is getting started.